libcontainer/cgroups/systemd/common.go

```diff
@@ -231,18 +230,21 @@ func systemdVersion(cm *dbusConnManager) int {
 	return version
 }
 
-func systemdVersionAtoi(verStr string) (int, error) {
-	// verStr should be of the form:
-	// "v245.4-1.fc32", "245", "v245-1.fc32", "245-1.fc32" (without quotes).
+func systemdVersionAtoi(str string) (int, error) {
+	// str should be of the form:
```

Comment:
Perhaps this would be useful to be put as GoDoc for the function as well.

Reply:
moved to godoc